test/test_mismatch_cli.py

```diff
@@ -93,3 +104,19 @@ def verify_db(test_db):
         "facebook",
     )
     assert expected in result
+
+
+@pytest.mark.skipif(not LONG_TESTS(), reason="Skipping long tests")
+def test_package_build(build_cleanup):
+    print("Building package")
+    build = subprocess.run(["python", "-m", "build"], check=True)
```

Comment:
Hello, To ensure the code runs on all systems, consider using the path to the Python executable, which can be accessed via `sys.executable` and `import sys`.

```suggestion
    build = subprocess.run([sys.executable, "-m", "build"], check=True)
```

Reply:
didn't know about this. thanks.